.circleci/config.yml

```diff
@@ -104,6 +117,125 @@ jobs:
+            echo 'export QEMU_AUDIO_DRV=none' >> $BASH_ENV
+            export PATH=$PATH:/usr/local/share/android-sdk/platform-tools/
+      - run:
+          name: Install Android sdk
+          command: |
+            HOMEBREW_NO_AUTO_UPDATE=1 brew tap homebrew/cask
+            HOMEBREW_NO_AUTO_UPDATE=1 brew cask install android-sdk
+      - run:
+          name: Install emulator dependencies
+          command: (yes | bash ./packages/mobile/scripts/install_android_emulator_image.sh > /dev/null) || true
+      - save_cache:
+          key: android-emulator-image-v3-{{ checksum "packages/mobile/scripts/install_android_emulator_image.sh" }}
+          paths:
+            - /usr/local/Caskroom/android-sdk/*
+      - run:
+          name: Install Android dependencies
+          command: |
+            HOMEBREW_NO_AUTO_UPDATE=1 brew install cask
+            HOMEBREW_NO_AUTO_UPDATE=1 brew cask install android-platform-tools
+            HOMEBREW_NO_AUTO_UPDATE=1 brew tap homebrew/cask-versions
+            HOMEBREW_NO_AUTO_UPDATE=1 brew cask install homebrew/cask-versions/adoptopenjdk8
+      - run: 
+          name: Creace Android Virtual Device (AVD)
+          command: |
+            avdmanager create avd -n Nexus_5X_API_28_x86 -k "system-images;android-26;google_apis;x86" -g google_apis -d "Nexus 5"
+            # Copy device configuarion, adv for some reason doesn't
+            mkdir -p ~/.android/avd/Nexus_5X_API_28_x86.avd/ && cp packages/mobile/e2e/conf/avd_conf.ini ~/.android/avd/Nexus_5X_API_28_x86.avd/config.ini
+      - run:
+          name: Set up nodejs
+          command: |
+            echo `. ~/.bash_profile` # for some mreason just `source ~/.bash_profile` makes the build fail
+            # install nvm
+            HOMEBREW_NO_AUTO_UPDATE=1 brew install wget
+            wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.0/install.sh | bash
+            echo -e '\nexport NVM_DIR="$HOME/.nvm"' >> ~/.bash_profile
+            echo -e '\n[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> ~/.bash_profile # add nvm to path
+      - run: echo `. ~/.bash_profile`
+      - run: nvm install v10.16.3 && nvm use v10.16.3
+      - run:
+          name: install miscellaneous
+          command: HOMEBREW_NO_AUTO_UPDATE=1 brew install tree
+      # Currently not used
+      # - run: npm install --global react-native-kill-packager 
+      - run:
+          # need to run this because it's another OS than install_dependecies job
+          name: yarn
+          command: |
+            # TODO, Make cache work and figure out why this doesn't work
+            # if [ ! -d node_modules ]; then
+            #     echo "No cache found, install yarn. Hold tight."
+            #     yarn
+            # else
+            #   echo "Cache found, just run post-script."
+            #   yarn postinstall
+            # fi
+            yarn
+            yarn build 
```

Comment:
any idea why this fails with 
```
tsc
src/index.ts(1,43): error TS2307: Cannot find module '@celo/contractkit'.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run build stderr:
warning package.json: No license field
error Command failed with exit code 2.

lerna ERR! yarn run build exited 2 in '@celo/dappkit'
```
if comment the line above and uncomment the if block @ashishb?

Reply:
No idea. Let's get the working code in and fix caching later.